setup.cfg

```diff
@@ -96,6 +96,8 @@ testing =
     coverage==4.5.3
     # send test coverage to codecov.io
     codecov==2.0.15
+    # use mock
+    backports.unittest_mock
```

Comment:
This shouldn't be used. Pytest way is to use fixtures and there's already `pytest-mock` integrated, it must be used for consistency.

Reply:
Done.